xds/src/test/java/io/grpc/xds/BootstrapperImplTest.java

```diff
@@ -39,13 +42,38 @@
 
 /** Unit tests for {@link Bootstrapper}. */
 @RunWith(JUnit4.class)
-public class BootstrapperTest {
+public class BootstrapperImplTest {
 
+  private static final String BOOTSTRAP_FILE_PATH = "/home/grpc-java/xds/src/test/bootstrap.json";
```

Comment:
Could the value be more obvious that it's a fake path? It looks like a real path in a docker image.

Reply:
Done.